components/ten_years_ago_component.py

```
@@ -0,0 +1,34 @@
+# 1. df = get_10years_ago_news 호출 : 10년 전 기사 데이터 가져옴
+# 2. con(df) or sidebarCon(df) : 가져온 데이터로 컴포넌트 생성
+
+def get_10years_ago_news():
+
+    correct_time = datetime.datetime.now() - relativedelta(years=10)
+    target_date = "2015-05-27"                          # 데이터가 없어서 날짜 부분 하드코딩
+    # target_date = correct_time.strftime('%Y-%m-%d')
+
+    # st.write(correct_time.strftime('%Y-%m-%d')) 
+
+    df = connectDB(f"select * from news_summary WHERE article_date ='{target_date}'" ) 
```

Ruff (0.11.9)

Undefined name `connectDB`

(F821)